system/netdata.service.v235.in

```diff
@@ -14,11 +14,11 @@ CacheDirectory=netdata
 StateDirectory=netdata
 LogsDirectory=netdata
 RuntimeDirectoryMode=0775
-StateDirectoryMode=0750
-CacheDirectoryMode=0750
-LogsDirectoryMode=0750
-PIDFile=netdata/netdata.pid
-ExecStart=@sbindir_POST@/netdata -P $PIDFILE -D
+StateDirectoryMode=0755
+CacheDirectoryMode=0755
+LogsDirectoryMode=2750
```

Comment:
Why are you changing the directories' permissions? Is 2750 correct for the logs directory?

Reply:
The SGID bit is almost certainly wrong here.

Reply:
Because that's what we are creating the directories with elsewhere. Its always been created with `2750` apparently. If this is wrong we should change it there and here.

Reply:
As mentioned, the SGID bit is absolutely wrong here (there is no reason whatsoever to put the SGID bit on a log directory). I'd argue that other than that it's probably correct though.

Reply:
Like I said, this is broken elsewhere, Just do a code search for this mode. Can you file a separate issue for this because I don't want to land mixed changes in case there are other roll-on effects on user systems :)